server.js

```diff
@@ -9,17 +9,35 @@ const server = express();
 // allow server to parse JSON bodies from POST/PUT/DELETE requests
 server.use(bodyParser.json());
 
+// TODO: DONE - write your server code here
 
+// get Bear model
+const Bear = require('./models');
 
+// POST /bears -- accepts bear
+server.post('/bears', (req, res) => {
+  const newBear = new Bear(req.body);
+  newBear.save()
+    .catch(error => res.status(500).json({ error }))
+    .then(result => res.json(result));
+});
 
+// GET /bears -- returns bears object
+server.get('/bears', (req, res) => {
+  Bear.find()
+    .catch(error => res.status(500).json({ error }))
+    .then(bears => res.json(bears.reduce((obj, bear) => {
+      obj[bear.id] = bear;
+      return obj;
+    }, {})));
+});
 
-
-// TODO: write your server code here
-
-
-
-
-
+// GET /bears/:id -- expect id parameter on url, return bear
+server.get('/bears/:id', (req, res) => {
+  Bear.findById(req.params.id)
```

Comment:
It is fine to pass the id in this way because Params will always just be params. and you're specifying which part of the params you want.